src/main.rs

```diff
@@ -3,10 +3,14 @@ mod server;
 
 use std::env;
 use std::net::SocketAddr;
+use tracing::{Level};
+use tracing_subscriber::FmtSubscriber;
 
 #[tokio::main]
 async fn main() {
-    tracing_subscriber::fmt::init();
+    let subscriber = FmtSubscriber::builder().with_max_level(Level::INFO).finish();
```

Comment:
This ignores any values from the `RUST_LOG` environment variable.  Can we change this to support both a default level of `INFO` and an override?  It would look something like:

```rust
use tracing_subscriber::{filter::LevelFilter, EnvFilter, FmtSubscriber};
// ...
let subscriber = FmtSubscriber::builder()
    .with_env_filter(
        EnvFilter::builder()
            .with_default_directive(LevelFilter::INFO.into())
            .from_env_lossy(),
    )
```

Reply:
I can't get it do unconditionally log the INFO level entries to stdout... What am I doing wrong?

Reply:
Not sure.  Here's the full patch I'm using.  Can you confirm this is what you've got and that `cargo run` still has no output?

```patch
diff --git a/src/main.rs b/src/main.rs
index 7d7f3f9..a15b25c 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -3,14 +3,18 @@ mod server;
 
 use std::env;
 use std::net::SocketAddr;
-use tracing::{Level};
-use tracing_subscriber::FmtSubscriber;
+use tracing_subscriber::{filter::LevelFilter, EnvFilter, FmtSubscriber};
 
 #[tokio::main]
 async fn main() {
-    let subscriber = FmtSubscriber::builder().with_max_level(Level::INFO).finish();
-    tracing::subscriber::set_global_default(subscriber)
-        .expect("setting default subscriber failed");
+    let subscriber = FmtSubscriber::builder()
+        .with_env_filter(
+            EnvFilter::builder()
+                .with_default_directive(LevelFilter::INFO.into())
+                .from_env_lossy(),
+        )
+        .finish();
+    tracing::subscriber::set_global_default(subscriber).expect("setting default subscriber failed");
 
     // TODO: Better arg parsing and error handling.
     let args: Vec<String> = env::args().collect();
```

Reply:
From my terminal:

```
httpbin-rs via 🦀 v1.69.0 
➜ unset RUST_LOG; cargo run -q &
[1] 26200

2023-04-26T19:59:09.936504Z  INFO httpbin: listening on http://127.0.0.1:8080

httpbin-rs via 🦀 v1.69.0 
✦ ➜ curl -so /dev/null http://localhost:8080/status/418 && kill %1 
[1]  + 26200 terminated  cargo run -q                                                                        

httpbin-rs via 🦀 v1.69.0 
➜ RUST_LOG=debug cargo run -q &                                                                
[1] 26418

2023-04-26T19:59:45.870564Z  INFO httpbin: listening on http://127.0.0.1:8080

httpbin-rs via 🦀 v1.69.0 
✦ ➜ curl -so /dev/null http://localhost:8080/status/418 && kill %1
2023-04-26T19:59:49.240869Z DEBUG hyper::proto::h1::io: parsed 3 headers
2023-04-26T19:59:49.240924Z DEBUG hyper::proto::h1::conn: incoming body is empty
2023-04-26T19:59:49.241231Z DEBUG hyper::proto::h1::io: flushed 398 bytes
2023-04-26T19:59:49.241352Z DEBUG hyper::proto::h1::conn: read eof
[1]  + 26418 terminated  RUST_LOG=debug cargo run -q     
```

Reply:
@MikeRalphson If you wanna move this out into a different issue, we can get the XML support merged!